sites/kit.svelte.dev/src/lib/search/search.js

```diff
@@ -3,17 +3,22 @@ import flexsearch from 'flexsearch';
 // @ts-expect-error
 const Index = /** @type {import('flexsearch').Index} */ (flexsearch.Index) ?? flexsearch;
 
+/**
+ * @typedef {{title:string, href:string, breadcrumbs:string[], content: string}} StoredBlock
+ */
```

Comment:
whither `Block`?

Reply:
i think creating a new type is overkill, we don't really need to store `title` separately https://github.com/sveltejs/kit/pull/8728